src/MergeContext.js

```diff
@@ -1010,6 +1013,7 @@ class PullRequest {
     }
 
     async _enterStaged(stagedStatuses) {
+        this._abandoned = false;
```

Comment:
This reset looks odd because it lacks the code to remove the previously set label (if any). Why do we not need to remove the label here?